website/docs/reference/resource-configs/redshift-configs.md

```diff
@@ -257,3 +257,90 @@ The workaround is to execute `DROP MATERIALIZED VIEW my_mv CASCADE` on the data
 </VersionBlock>
 
 </VersionBlock>
+
+<VersionBlock firstVersion="1.8">
+
+## Unit test limitations
+
+Redshift doesn't support Unit tests when the SQL in the common table expression (CTE) contains functions such as `LISTAGG`, `MEDIAN`, `PERCENTILE_CONT`, etc. These functions must be executed against a user-created table. dbt combines given rows to be part of the CTE, which Redshift does not support. 
```

Comment:
@matthewshaver What I like about how the content is [framed below](https://github.com/dbt-labs/docs.getdbt.com/pull/5508/files#diff-5d2cee8c48606c33956458e7b89eacbda29c62349ad49d68ea9380173acda697R339-R343) is that it's clearer than this paragraph and it's framed in the positive. I think we should rewrite this paragraph so we: 
1. Communicate that unit test are limited by actually saying what they need to do to get unblocked.  like "Avoid using SQL in CTE functions such as.." and "Unit tests can be used when you.."
2. Explain how to overcome this limitation. Instead of the passive language of "These functions must be executed..." Use more active language like "Execute these functions against..."